tests/test_ace.rb

```diff
@@ -118,11 +126,11 @@ def create_destroy_ace(afi, entry)
     assert(found,
            "#{afi_cli} acl #{acl_name} seqno #{@seqno} is not configured")
 
```

Comment:
Instead of removing the entry check how about this instead?

```
action = entry.include?(:action) ? entry[:action] : entry[:remark]
assert_show_match(pattern: /\s+#{@seqno} #{action} .*$/,
                                 msg:     "failed to create ace seqno #{@seqno}")
```